paddle/operators/sequence_conv_op.cc

```diff
@@ -128,25 +128,25 @@ class SequenceConvOpMaker : public framework::OpProtoAndCheckerMaker {
         "this LoDTensor is a matrix with shape (T, D), where, T is the "
         "total time steps in this mini-batch, D is the output feature size.");
 
-    AddAttr<bool>("padding_trainable",
+    AddAttr<bool>("paddingTrainable",
                   "(bool, default false) the padding data of SequenceConvOp "
                   "is trainable or not.")
         .SetDefault(false);
-    AddAttr<int>("context_length",
-                 "(int, default 3) the context_length of SequenceConvOp is the "
+    AddAttr<int>("contextLength",
+                 "(int, default 3) the contextLength of SequenceConvOp is the "
                  "height of the convolution kernel.")
         .SetDefault(3)
         .GreaterThan(0);
-    AddAttr<int>("context_start",
-                 "(int, default 0) the context_start of SequenceConvOp "
+    AddAttr<int>("contextStart",
+                 "(int, default 0) the contextStart of SequenceConvOp "
                  "represents the beginning of the convolution of the number of "
                  "rows of sequence, which can be negative.")
```

Comment:
More explanations:

The negative number means to pad contextStart time-steps of zeros or learnable parameters at the beginning of each instance. The positive number means to skip contextStart time-steps of each instance.